pkg/workflows/sdk/gen/compute.go.tmpl

```diff
@@ -6,6 +6,39 @@ import (
     "github.com/smartcontractkit/chainlink-common/pkg/values"
 )
 
+type ComputeConfig[C any] struct {
```

Comment:
This struct can go in compute.go, since we don't need more than one of them, there's no need to generate it :).